suite-native/module-accounts/src/components/AccountDetail.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+    AccountsStackParamList,
+    AccountsStackRoutes,
+    Screen,
+    ScreenHeader,
+    StackProps,
+} from '@suite-native/navigation';
+import {
+    AccountsRootState,
+    fetchTransactionsThunk,
+    selectAccountName,
+    selectAccount,
+} from '@suite-common/wallet-core';
+import { Box, Text } from '@suite-native/atoms';
+import { TransactionList } from '@suite-native/module-transactions';
+
+import { AccountBalance } from './AccountBalance';
+
+export const AccountDetail = ({
+    route,
+}: StackProps<AccountsStackParamList, AccountsStackRoutes.AccountDetail>) => {
+    const { accountKey } = route.params;
+    const account = useSelector((state: AccountsRootState) => selectAccount(state, accountKey));
+    const accountName = useSelector((state: AccountsRootState) =>
+        selectAccountName(state, accountKey),
+    );
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        if (!account) return;
+        dispatch(
+            fetchTransactionsThunk({
+                account,
+                page: 1,
+                perPage: 25,
+            }),
+        );
+    });
+
+    if (!account) return null;
```

Comment:
The `selectAccount` selector uses `.find` function which returns the account or undefined if that fails. Technically speaking, the account should always be there unless something really weird happens, but since TS can return undefined, this check needs to be here to avoid doing `account?.<something>` everywhere below.